tuning-fork/boilerplate
# React query/implement react query part two

## 🤺 Summary
This PR continues #337 - Adding react-query to all components/pages, specifically mutations. 

Steps to test:
Manual test CRUD routes as normal; check to see that react-query implementation is working.
